README.md

```diff
@@ -435,7 +437,7 @@ test('passes when value is a function', () => {
 Use `.toThrowWithMessage` when checking if a callback function throws an error with a given error type and given error message. Message can either be a `String` or a `RegExp`.
 
 ```js
-test('throws an error of type TypeError with message "hello world"', () => {
+test('throws an error of type TypeError with message "hello world"', async () => {
```

Comment:
Can you make this a separate test instead?

Reply:
Applied in a05d5ce87af5a9b57ae93a677a1e5bc098047fd3